[CM-1270] Add dependency section in read me (#6)
yml-org/ycarousel-ios: README.md

present(carouselViewController, animated: true, completion: nil)
```

Dependencies
----------

> Y—Carousel depends upon our [Y—CoreUI](https://github.com/yml-org/ycoreui) and [Y—MatterType](https://github.com/yml-org/ymattertype) frameworks (both also open source and Apache 2.0 licensed).

Reviewer: ```suggestion
Y—Carousel depends upon our [Y—CoreUI](https://github.com/yml-org/ycoreui) framework (which is also open source and Apache 2.0 licensed).
```
